contrib/drivers/lvm/metricsCLI.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2019 Huawei Technologies Co., Ltd. All Rights Reserved.
```

Comment:
I prefer to change the file name to `metricscli.go` or `metrics_cli.go`

Reply:
Done, please check